doc/install_basic.txt

```diff
@@ -0,0 +1,47 @@
+.. _install_basic:
+
+Basic user install instructions
+===============================
+
+These are the 3 simple ways of obtaining Theano, and additional requirements
+will change depending on the way you choose.
+
+pip
+~~~
+    
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. Use the [sudo] or [--user] option depending on your admin rights.
+
+.. code-block:: bash
+
+    [sudo] pip install Theano [--user]
+
+conda
+~~~~~
+
+say something about conda and miniconda
+
+Anaconda
+~~~~~~~~
+
+If you don't want to bother with anything, Theano can be acquired through
+Anaconda_.
+
+
+.. _pip: http://pypi.python.org/pypi/pip
+.. _anaconda: https://docs.continuum.io/
+
+.. _testing_installation:
+
+Testing your installation
```

Comment:
Yes I was talking about the testing only. Is it really installation dependent? I would have said it's not, we can maybe discuss it IRL. I agree that this is the right place to keep it if it's installation dependent, but in case it's not I'd rather merge and move all the testing instructions in a separate page or in a section in some of the existing pages (but I would be inclined toward the first option).